forge/test/model_demos/high_prio/cnn/pytorch/test_detr.py

```diff
@@ -0,0 +1,69 @@
+# SPDX-FileCopyrightText: © 2024 Tenstorrent AI ULC
+
+# SPDX-License-Identifier: Apache-2.0
+# Detr model having both object detection and segmentation model
+# https://huggingface.co/docs/transformers/en/model_doc/detr
+
+import pytest
+import forge
+import requests
+from torchvision import transforms
+from transformers import (
+    DetrForObjectDetection,
+    DetrImageProcessor,
+    DetrConfig,
+    DetrFeatureExtractor,
+    DetrForSegmentation,
+)
+from PIL import Image
+
+detection_variants = ["facebook/detr-resnet-50"]
+segmentation_variants = ["facebook/detr-resnet-50-panoptic"]
+
+# Load the image
+url = "http://images.cocodataset.org/val2017/000000397133.jpg"
+image = Image.open(requests.get(url, stream=True).raw).convert("RGB")
+
+
+@pytest.mark.parametrize("variant", detection_variants)
+def test_detr_detection(variant):
+
+    # Load the model
+    model = DetrForObjectDetection.from_pretrained(variant)
+    processor = DetrImageProcessor.from_pretrained(variant)
+    model_config = DetrConfig.from_pretrained(variant)
+
+    # Preprocess the image for the model
+    inputs = processor(images=image, return_tensors="pt")
+    input_tensor = transforms.ToTensor()(image)
+    input_batch = input_tensor.unsqueeze(0)
+
+    # Sanity test
+    fw_out = model(input_batch)
+
+    # Compiler test
+    compiled_model = forge.compile(
+        model, sample_inputs=[input_batch], module_name="pt_" + str(variant.split("/")[-1].replace("-", "_"))
+    )
+    # AttributeError: <class 'tvm.ir.op.Op'> has no attribute name_hint
+
+
+@pytest.mark.parametrize("variant", segmentation_variants)
+def test_detr_segmentation(variant):
+    feature_extractor = DetrFeatureExtractor.from_pretrained(variant)
```

Comment:
If you decide to remove `inputs` and keep the `input_tensor`, you should also remove this and `DetrFeatureExtractor` from imports